src/main/scala/ayai/systems/ai/pathfinding/PathfindingSystem.scala

```diff
@@ -25,18 +25,23 @@ class PathfindingSystem(actorSystem: ActorSystem) extends EntityProcessingSystem
     val map = GraphFactory.generateGraph(roomWorld)
 
     for {
-      pathfinding <- e.getComponent(classOf[AStar]).toList.map(_.asInstanceOf[AStar])
+      pathfinding <- e.getComponent(classOf[AStarPathfinder]).toList.map(_.asInstanceOf[AStarPathfinder])
       position <- e.getComponent(classOf[Position]).toList.map(_.asInstanceOf[Position])
     } {
-//      println(s"Processing entity = ${e.uuid} for AStar at position $position")
+      //println(s"Processing entity = ${e.uuid} for AStarPathfinder at position $position")
 
       // TODO: replace with real target
-      pathfinding.findPath(map, position, position) match {
+      val movementStyle = new ManhattanMovementStyle
+      val graphView = new NodeMatrixGraphView(map, movementStyle)
+      val conversionRatio: Float = ???
```

Comment:
Well, the conversion *to* `NodeMatrixPosition` happens in `GraphFactory`, and looks like this:

```scala
val gridPosition = NodeMatrixPosition.fromPosition(p, tileMap.tileSize.toFloat)
```

So you need to somehow look up which `tileMap` the current entity is on. Then just call `tileMap.tileSize.toFloat`.